packages/utilities/src/date-time/dateMath/DateMath.ts

```diff
@@ -0,0 +1,418 @@
+import { DayOfWeek, MonthOfYear, FirstWeekOfYear, DateRangeType } from '../dateValues/DateValues';
```

Comment:
So you're copy/pasting these and some other utilities, but aren't removing the previous versions yet- what's the end goal? Is office-ui-fabric-react and date-time packages going to reference the utilities package to get these values out?

The original intention of the date-time package was to have all the date related logic wrapped up into it, so that we didn't have it spread around to multiple packages. At the moment, date-time just relies on OUFR's date logic, but the long term goal is to move all the logic out of OUFR and into date-time. 

With this solution, we end up spreading date logic into the utilities package. Is north star trying to avoid a dependency on date-time package as well as the OUFR package? Or should we take another look at getting that logic out of OUFR entirely so you can pull in just date-time without pulling in all of OUFR as well?